src/grafana/common/zarf.yaml

```diff
@@ -19,7 +19,7 @@ components:
           - ../chart/values.yaml
       - name: grafana
         url: https://grafana.github.io/helm-charts/
-        version: 8.11.0
```

Comment:
This change might be beneficial as we use sidecars. See:

* https://github.com/defenseunicorns/uds-core/blob/ee55ab10091f1ef150bc1050b19ca8ec494deaa8/src/grafana/values/values.yaml#L7
* https://github.com/grafana/helm-charts/pull/3585/files